h/accounts/test/views_test.py

```diff
@@ -34,9 +34,10 @@ def __init__(self, type_, active):
 
 class FakeUser(object):
     def __init__(self, **kwargs):
-        self.activation = None
-        for k in kwargs:
-            setattr(self, k, kwargs[k])
+        defaults = {'activation': None, 'email': None, 'password': None}
```

Comment:
These changes are to simplify the changed `ProfileController` tests later on in this file.